src/data-access/users.ts

```diff
@@ -0,0 +1,67 @@
+import { randomUUID } from 'crypto';
```

Comment:
I would recommend putting all business logic in a business logic-related class, like UserService. It the future it can be easily extended with DB repo class for example

Reply:
Applied needs re-review.